components/dashboard/src/components/DropDown2.tsx

```diff
@@ -207,3 +240,40 @@ export const DropDown2: FunctionComponent<DropDown2Props> = (props) => {
         </div>
     );
 };
+
+type DropDown2SelectedElementProps = {
+    iconSrc: string;
+    loading?: boolean;
+    title: ReactNode;
+    subtitle: ReactNode;
+    htmlTitle?: string;
+};
+
+export const DropDown2SelectedElement: FC<DropDown2SelectedElementProps> = ({
+    iconSrc,
+    loading = false,
+    title,
+    subtitle,
+    htmlTitle,
+}) => {
+    return (
+        <div className={classNames("flex items-center", loading && "animate-pulse")} title={htmlTitle}>
```

Comment:
**suggestion(non-blocking)**: It'd be better to _not_ use skeleton loaders for static content, like the icons here. However, we could think about these details in the next iteration of the new workspace page.